metadata-ingestion/src/datahub/utilities/mapping.py

```diff
@@ -171,30 +194,12 @@ def get_operation_value(
         operation_config: Dict,
         match: Match,
     ) -> Optional[Union[str, Dict, List[str]]]:
-        def _get_best_match(the_match: Match, group_name: str) -> str:
-            result = the_match.group(0)
-            try:
-                result = the_match.group(group_name)
-                return result
-            except IndexError:
-                pass
-            try:
-                result = the_match.group(1)
-                return result
-            except IndexError:
-                pass
-            return result
-
-        match_regexp = r"{{\s*\$match\s*}}"
-
         if (
             operation_type == Constants.ADD_TAG_OPERATION
             and operation_config[Constants.TAG]
         ):
             tag = operation_config[Constants.TAG]
-            tag_id = _get_best_match(match, "tag")
-            if isinstance(tag_id, str):
```

Comment:
This if condition is missing in refractored code but also it does not seem required, as  match object essentially returns a string if single argument is passed - https://docs.python.org/3/library/re.html#re.Match.group

Reply:
Yep the get_best_match method always returned a string so it wasn't necessary